db/migrate/20150427002238_add_sort_order_and_featured_content_to_organizations.rb

```diff
@@ -0,0 +1,6 @@
+class AddSortOrderAndFeaturedContentToOrganizations < ActiveRecord::Migration
+  def change
+    add_column :organizations, :sort_order,       :integer, null: false, default: 0
+    add_column :organizations, :featured_content, :string,  null: false, default: ''
```

Comment:
Line is too long. [84/80]<br>Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.